js/components/scrubberBar.js

```diff
@@ -196,7 +196,7 @@ var ScrubberBar = React.createClass({
       backgroundColor: this.props.skinConfig.controlBar.scrubberBar.backgroundColor
     };
     var bufferedIndicatorStyle = {
-      width: (parseFloat(this.props.buffered) / parseFloat(this.props.duration)) * 100 + "%",
+      width: Math.min((parseFloat(this.props.buffered) / parseFloat(this.props.duration)) * 100, 100) + "%",
```

Comment:
Yup, it'll work, I need to use the same logic as the playhead indicator - https://github.com/ooyala/html5-skin/blob/stable/js/components/scrubberBar.js#L203